tests/dummy/config/environment.js

```diff
@@ -40,7 +40,7 @@ module.exports = function(environment) {
     ENV.baseURL = '/';
     ENV.locationType = 'none';
 
-    ENV.EmberENV.RAISE_ON_DEPRECATION = true;
+    ENV.EmberENV.RAISE_ON_DEPRECATION = false;
```

Comment:
i guess it's better to have deprecations turned on to fix problems before they make their way into apps